### DISCUSSION TITLE
Amazon forcing to download in .KFX format for certain kindle content

### DISCUSSION THREAD

**Discussion by APGHOSH** [apprenticeharper/DeDRM_tools#1996] [Q&A]
The issue is with the 2 Amazon books below that I have purchased.

https://www.amazon.in/Burning-Megha-Majumdar-ebook/dp/B089QMGFGN/ref=sr_1_1?crid=2W09P4WHIEV9O&keywords=a+burning+megha+majumdar&qid=1642489317&sprefix=a+burning%2Caps%2C305&sr=8-1

https://www.amazon.in/Whereabouts-Novel-Jhumpa-Lahiri-ebook/dp/B091D4LM56/ref=sr_1_1?crid=C2S6XC2696VW&keywords=whereabouts+jhumpa+lahiri&qid=1642489339&sprefix=whereabouts%2Caps%2C288&sr=8-1

These 2 books force me to use the latest version of Kindle for PC. And it seems that they only support download in .KFX format. I changed the registry key value not to support KFX but then it would no longer download and urge me to download the latest version even though I already had the latest version.

The screenshots are attached below.

![Kindle for PC](https://user-images.githubusercontent.com/1394703/149889157-2aaeee79-754b-4664-86d5-ebd39f18c5aa.jpg)

![Registry key](https://user-images.githubusercontent.com/1394703/149889186-067ebe33-62e4-42b3-8d56-6cd8fd35857b.jpg)

Can you please advise what should be done? Thanks in advance and any help would be greatly appreciated.

**Comment by APGHOSH** [comment]
These books can be downloaded when the registry variable is set to true i.e. only when the KFX download is allowed.

**Comment by j-howell** [comment]
It was noticed in March of last year that some English language books published in India are only available in KFX format using DRM that cannot be removed by these tools. There is currently no DRM removal solution for these book.

This restriction is also applies to expensive textbooks from some publishers. It is likely an added anti-piracy measure.

**Reply by Dragoon91786** [reply]
@j-howell,

Does this include the GitHub project, [**NoDRM**](https://github.com/T3rry7f/NoDRM) by @T3rry7f that @apprenticeharper mentioned in the [**about**](https://github.com/apprenticeharper/DeDRM_tools#no-longer-maintained) notice?

**Reply by ElleKayEm** [reply]
@Dragoon91786 Harper was referring to this: https://github.com/noDRM/DeDRM_tools

**Reply by j-howell** [reply]
@Dragoon91786 The project you mentioned has nothing to do with e-book DRM.

ElleKayEm provided a link to the fork of this project that is currently being maintained. What I wrote above applies to the latest version from that fork.
